sites/docs/src/content/docs/guidelines/components/modules.md

```diff
@@ -464,7 +464,30 @@ tuple val(meta), path('*.tab'), emit: tab,  optional: true
 
 ### One output channel per output file type
 
-Each output file SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+Each output file type SHOULD be emitted in its own channel (and no more than one), along with the `meta` map if provided ( the exception is the versions.yml ).
+
+In some cases the file format can be different between files of the same type (e.g. indices: `.bai` and `.crai`). These different file formats SHOULD be part of the same output channel since they are the same file type.
+
+```groovy
+tuple val(meta), path("*.{bai,crai}"), emit: index
+```
+
+:::info{title="Rationale" collapse}
+This approach simplifies the process of retrieving and processing specific types of output, as each type can be easily identified and accessed within its designated channel.
+
+So when the output definition of module called `TXT_MODULE` looks like this:
+
+```groovy
+tuple val(meta), path("*.txt"), emit: txt
+```
+
+The output files can be accessed like this:
+
+```groovy
+TXT_MODULE.out.txt
+```
```

Comment:
I don't really get this example... usn't this just describing output channels?

Reply:
Yes I'm not sure what would be a better example here. Do you have any ideas?

Reply:
Good idea! On it!

Reply:
@jfy133 I updated the example, is this better?